packages/dap/dap.1.0.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Debug adapter protocol"
+description: """
+The Debug Adapter Protocol defines the protocol used between an editor or IDE and a debugger or runtime.
+"""
+maintainer: "文宇祥 <hackwaly@qq.com>"
+authors: "文宇祥 <hackwaly@qq.com>"
+license: "MIT"
+homepage: "https://github.com/hackwaly/ocaml-dap"
+bug-reports: "https://github.com/hackwaly/ocaml-dap/issues"
+dev-repo: "git://git@github.com:hackwaly/ocaml-dap.git"
+doc: "https://hackwaly.github.io/ocaml-dap/"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {build & >= "1.3"}
```

Comment:
```suggestion
  "dune" {>= "1.3"}
```

Dune doesn't support the build tag as it's not forward compatible

Reply:
@hackwaly I had sent a PR for this during the last release since it was an issue of the first two PRs, see https://github.com/hackwaly/ocaml-dap/pull/1 can you merge it?

Reply:
And the dune-project file requires 2.7 ```suggestion
  "dune" {>= "2.7"}
```
You need to update the Lang requirement in the dune-project file to support older versions of dune

Reply:
Thanks a lot